org.eclipse.scout.rt.rest/src/main/java/org/eclipse/scout/rt/rest/client/chunked/IChunkedDataReader.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) 2010, 2025 BSI Business Systems Integration AG
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.scout.rt.rest.client.chunked;
```

Comment:
How important is it that this interface is in the "client" package? If not so much, I would just put it in the "chunked" package with the writer interface.